packages/system/src/generator.ts

```diff
@@ -50,40 +50,46 @@ export class StyleGenerator {
         !/^\w+\(.*\)$/.test(propValue) // exclude CSS functions
       );
     };
-
-    for (const [propName, propValue] of Object.entries(props)) {
+    /* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-return -- FIXME */
+    const convertStyle = (name: string, value: any) => {
```

Comment:
Can you add JSDoc and an @example tag for this function? Since the return type is any, it would at least make it easier for future readers to understand what the expected return value is. 🙏